docs/getting-started.md

```diff
@@ -193,7 +193,7 @@ Node comes with npm, which lets you install the React Native command line interf
 Run the following command in a Terminal:
 
 ```
-npm install -g react-native-cli
+npm install @react-native-community/cli
```

Comment:
this doesn't need to be installed. It's available under `react-native` binary:
```
yarn react-native
npx react-native
```
will defer to the CLI, so you can do `yarn react-native info` and other commands. Please remove this step about global installation